integrations/artifactory_buildinfo.rst

```diff
@@ -0,0 +1,63 @@
+.. _integrations_artifactory_build_info:
+
+Artifactory Build Info
+======================
+
+The `Artifactory build info <https://www.buildinfo.org/>`_ Artifactory build info is a recollection of the metadata of a build.
+This json-formatted file includes all the details about the build broken down into segments like version history, artifacts, project
```

Comment:
I'm not sure about the version history, is that covered by the BuildInfo?

Reply:
not sure about it. I got it from https://www.buildinfo.org/

Reply:
not sure about it. I got it from https://www.buildinfo.org/